Tasks/Maven/pmdForMaven.ts

```diff
@@ -1,9 +1,54 @@
 /// <reference path='../../definitions/vsts-task-lib.d.ts' />
 
+import util = require('util');
+import path = require('path');
+import fs = require('fs');
+import xml2js = require('xml2js');
```

Comment:
Please add a comment on how you're importing this function once the build team replies